dcp_prototype/backend/wrangling/migrations/scripts/migrate_dcp_one_to_dcp_two.py

```diff
@@ -96,8 +101,8 @@ def consume_file(prefix, bucket, filequeue, dataset_metadata):
         qsize = filequeue.qsize()
         if qsize % 1000 == 0 and qsize > 0:
             print(f"Working with JSON input file: {file_prefix}, queued size={qsize}")
-        object = S3_RESOURCE.Object(bucket, file_prefix)
-        object_body = object.get()["Body"].read()
+        object = s3_client.get_object(Bucket=bucket, Key=file_prefix)
```

Comment:
Given this change, I believe you can remove the `S3_RESOURCE` global object from line 27.

Reply:
removed